data/tlg1286/tlg001/tlg1286.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,543 @@
+                       <editor>Walter Scott</editor>
+                       <imprint>
+                           <pubPlace>Oxford</pubPlace>
+                           <publisher>Oxford University Press</publisher>
+                           <date>1924</date>
+                       </imprint>
+                   </monogr>
+                   <ref target="https://archive.org/details/ScottHermeticaVolOne/page/n117/mode/2up">The Internet Archive</ref>
+               </biblStruct>
+           </sourceDesc>
+       </fileDesc>
+       <encodingDesc>
+           <editorialDecl>
+               <correction>
+                   <p>Lace v. 0.6.11 copyright 2013-2020, Bruce Robertson, Dept. of Classics, Mount Allison University. Developed through the support of the <ref target="https://nbif.ca/en">NBIF</ref>.
+                   </p>
+               </correction>
+           </editorialDecl>
+           <p>The following text is encoded in accordance with EpiDoc standards and with the CTS/CITE Architecture</p><!-- sample top level refsDecl for lines -->
+           <refsDecl n="CTS">
+               <cRefPattern matchPattern="(\w+)" n="sections" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"/>
+           </refsDecl><p>This pointer pattern extracts sections.</p>
+       </encodingDesc>
+       <profileDesc>
+           <langUsage>
+               <language ident="grc">Greek</language>
+               <language ident="lat">Latin</language>
+           </langUsage>
+       </profileDesc>
+       <revisionDesc><!-- put your name in 'who' and this date in YYYY-MM-DD format in 'when' -->
+           <change when="2020-02-02" who="Jane Smith">initial markup of new file.</change>
+           <change when="2020-10-15" who="Charlotte Houghton">CTS conversion.</change>
+       </revisionDesc>
+   </teiHeader>
+    <text>
+        <body>
+            <div type="edition" n="urn:cts:greekLit:tlg1286.tlg001.1st1K-grc1" xml:lang="grc">
+                    <div type="textpart" subtype="section" n="1">
+                        <pb facs="ScottHermeticaVolOne_0117"/>
+                        <p rend="indent">
+                        Ἐννοίας μοί ποτε γενομένης περὶ τῶν ὄντων, καὶ μετεωρισθείσης 
+                        μὲN τῆς διανοίας σφόδρα, ⟨⟨ὕπνῳ⟩⟩ ⟨δὲ⟩ κατασχεθεισῶν 
+                        μου τῶν σωματικῶν αἰσθήσεων, ⟨οὐ μέντοι⟩ καθάπερ 
+                        ⟨τ⟩οῖ⟨ς⟩ [[ὕπνῳ]] βεβαρημένοι⟨ς⟩ ἐκ κόρου τροφῆς ἢ ἐκ κόπου <l n="5"/> 
+                        σώματος, ἔδοξά τινα ὑπερμεγέθη μέτρῳ ἀπεριορίστῳ ⟨ἐν⟩τνγχάνοντα 
+                        καλεῖν μου τὸ ὄνομα, καὶ λέγεν [τα] μοι· 
+                        Τί βούλει ἀκοῦσαι καὶ θεάσασθαι, καὶ νοήσας μαθεῖν καὶ 
+                        γνῶναι; — φημὶ ἐγώ· 
+                        </p>
+                    </div>
+                <div type="textpart" subtype="section" n="2">
+                    <p rend="merge">
```

Comment:
`<p rend="merge">` not used. Simple `<p>` suffices when indentation is not used.